public/locales/en/translation.json

```diff
@@ -1,17 +1,20 @@
 {
-  "welcome": {
-    "message": "Welcome to React"
+  "whoAreWe": {
+    "header": "WHO ARE WE?"
   },
-  "React": {
-    "1": "HEY GUUYS"
+  "aboutPart": {
```

Comment:
1. things should be 0 indexed
2. in general we should avoid using non-descriptive numeric keys like this. plus, i'm pretty sure i18n accepts a list, so if you were going to use "0", "1," 2", ... a json list should just be used here.
3. i don't see aboutPart used in this change. we try to avoid coupling changes with unused code, particularly because what often happens is that people forget they added it or things change, and it ends up just being filler code in the codebase. same with `"gunsu"` . if it's not being used in the same PR, delete it.

Reply:
we'll be changing 1's and 0's. i will delete them.

Reply:
> we'll be changing 1's and 0's. i will delete them.

instead of changing and deleting them, as Louis said, can we use a list like this:
"aboutPart": [
       "We are",   ".",   ".",   "."
]